docs/user_guide/thresholder.rst

```diff
@@ -0,0 +1,560 @@
+diabetes whose stay lasted one to fourteen days. We would like to develop a
+classification model, which decides whether the patients should be suggested
+to their primary care physicians for an enrollment into the high-risk care
+management program. The positive prediction will mean recommendation into the
+care program.
+
+
+.. doctest:: thresholder
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> import pandas as pd
+    >>> import numpy as np
+    >>> from sklearn.model_selection import train_test_split
+    
+    >>> #Get dataset, drop columns with unknown sf values
+    >>> df = pd.read_csv("https://raw.githubusercontent.com/fairlearn/talks/main/2021_scipy_tutorial/data/diabetic_preprocessed.csv")
+
+    >>> df = df[df['gender'] != 'Unknown/Invalid']
+    >>> df = df[df['race'] != 'Unknown']
+
+    >>> Y, A = df.loc[:, 'readmit_30_days'], df.loc[:, 'race']
+
+    >>> #Create dummy's, drop A from X, drop columns that correlate with Y 
+    >>> X = pd.get_dummies(df.drop(columns=["race", "discharge_disposition_id",
+    ...                                     "readmitted",
+    ...                                     "readmit_binary",
+    ...                                     "readmit_30_days"]))
+
+    >>> X_train, X_test, Y_train, Y_test, A_train, A_test, df_train, df_test = train_test_split(
+    ...        X,
+    ...        Y,
+    ...        A,
+    ...        df,
+    ...        test_size=0.50,
+    ...        stratify=Y,
+    ...        random_state=445)
+
+    >>> #Resample datset such that training data has same number of positive and negative samples
+    
+    >>> def resample_dataset(X_train, Y_train, A_train):
+    >>>   negative_ids = Y_train[Y_train == 0].index
+    >>>   positive_ids = Y_train[Y_train == 1].index
+    >>>   np.random.seed(1)
+    >>>   balanced_ids = positive_ids.union(np.random.choice(a=negative_ids, size=len(positive_ids)))
+
+    >>>   X_train = X_train.loc[balanced_ids, :]
+    >>>   Y_train = Y_train.loc[balanced_ids]
+    >>>   A_train = A_train.loc[balanced_ids]
+    
+    >>>   return X_train, Y_train, A_train
+
+    >>> X_train, Y_train, A_train = resample_dataset(X_train, Y_train, A_train)
+   
+
+Classifier: 'Neutral' predictions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The classifier we will be using throughout this example is a
+RandomForestClassifier. We first investigate the output when we call predict
+on this estimator, without using :class:`Thresholder`. This means all groups
+are thresholded at 0.5 (this is the default threshold used by most
+classification algorithms). 
+
+Considering the nature of the problem at hand, important aspects of our model
+to investigate are:
+
+- How many patients are recommended for care (selection rate = what % of predictions are positive) 
+- Reducing false negative rate (we don't want people that need extra care not receiving it)
+
+.. doctest:: thresholder
+    :options:  +NORMALIZE_WHITESPACE
```

Comment:
Re: your question:
Yes, this is how you can start code cells that you want executed and where you want the output to be checked. The whitespace option is needed to avoid weird issues with formatting when you output tables. You can remove it and see what happens. It might start complaining... Certainly fine to add it!